src/Forms/Shared/Samples/Symbology/UseDistanceCompositeSym/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Name": "Render a graphic with a distance composite scene symbol",
```

Comment:
The common samples' name is "Distance Composite Symbol" just keeps the name shorter as "Render a graphic with a distance composite scene symbol" could be a bit long for mobile device UIs